contracts/FixedProductMarketMaker.sol

```diff
@@ -276,12 +298,31 @@ contract FixedProductMarketMaker is ERC20, ERC1155TokenReceiver {
 
         conditionalTokens.safeTransferFrom(msg.sender, address(this), positionIds[outcomeIndex], outcomeTokensToSell, "");
 
-        uint feeAmount = returnAmount.mul(fee) / (ONE.sub(fee));
-        mergePositionsThroughAllConditions(returnAmount);
+        uint protocolFeeAmount;
+        uint lpFeeAmount;
```

Comment:
atm `lpFeeAmount` is being assigned interesting info about lp earnings on line 314, but the variable is not actually being used anywhere. We could maybe remove it or add an `lpFeeAmount` field to the `FPMMBuy` and `FPMMSell` events